cmd/istioctl/main.go

```diff
@@ -318,10 +332,14 @@ Example usage:
 func init() {
 	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "c", "",
 		"Use a Kubernetes configuration file instead of in-cluster configuration")
-	rootCmd.PersistentFlags().StringVarP(&managerAddr, "managerAddr", "m", "",
-		"Set your Istio manager address")
 	rootCmd.PersistentFlags().StringVarP(&namespace, "namespace", "n", api.NamespaceDefault,
 		"Select a Kubernetes namespace")
+	rootCmd.PersistentFlags().StringVar(&istioNamespace, "istioNamespace", defaultIstioNamespace,
+		"Namespace where Istio system resides")
+	rootCmd.PersistentFlags().StringVar(&istioManagerService, "managerAPIService", "istio-manager:8081",
+		"Name of istio-manager service")
```

Comment:
The meaning of this flag is different based on --kube. Per https://kubernetes.io/docs/concepts/cluster-administration/access-cluster/#discovering-builtin-services it is the service name + port. When --kube=false it is the server address. I updated the flag doc to reflect this.